libfwupd/fwupd-self-test.c

```diff
@@ -426,6 +428,10 @@ fwupd_device_func (void)
 		"        \"deadbeef\"\n"
 		"      ],\n"
 		"      \"Size\" : 1024,\n"
+		"      \"Locations\" : [\n"
+		"        \"http://foo.com\",\n"
+		"        \"https://foo.com\"\n"
```

Comment:
since you mention in the changelog that you can have some non-`http(`s)` prefixed URLs, probably should make sure to include one here too.

Reply:
Done, thanks.